frontend/src/main/components/OurTable.js

```diff
@@ -95,7 +95,7 @@ export function ButtonColumn(label, variant, callback, testid, displayText) {
         data-testid={`${testid}-cell-row-${cell.row.index}-col-${cell.column.id}-button`}
         className={cell.column.id} 
       >
-        {displayText}
+        {displayText || label}
```

Comment:
I dont prefer changes to OurTable unless absolutely necessary. but I see that this is probably to fix the change introduced in a previous PR that maybe shouldn't have been allowed but I'll let it slide this time. So long as all the tables are still working as intended.